src/main/java/com/klarna/hiverunner/HiveServerContainer.java

```diff
@@ -184,6 +191,11 @@ public HiveConf getHiveConf() {
         return hiveServer2.getHiveConf();
     }
 
-
+    public VariableSubstitution getVariableSubstitution() {
```

Comment:
It's not here in the diff but there are other public methods in this class where you call new VariableSubstitution() without this guard. I'd guess you want to replace all calls to such instantiation with a call to this method instead?